src/sentry/api/endpoints/organization_dashboard_widget_details.py

```diff
@@ -0,0 +1,87 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry.api.base import DocSection
+from sentry.api.bases.dashboard import (
+    OrganizationDashboardWidgetEndpoint, WidgetSerializer
+)
+from sentry.api.serializers import serialize
+from sentry.models import ObjectStatus
+
+
+class OrganizationDashboardWidgetDetailsEndpoint(OrganizationDashboardWidgetEndpoint):
+
+    doc_section = DocSection.ORGANIZATIONS
+
+    def get(self, request, organization, widget):
```

Comment:
@billyvg do we need a get method for widgets if there is a get method on dashboard?